api-reference/beta/resources/security_overview.md

```diff
@@ -0,0 +1,46 @@
+# Use Microsoft Graph to integrate with the Intelligent Security Graph
+

```

Comment:
How about the title "Use the security API to integrate with the Microsoft Intelligent Security Graph"?